recipe/bld.bat

```diff
@@ -1,22 +1,42 @@
-robocopy . "%PREFIX%" /S /XF bld.bat
-if errorlevel NEQ 1 exit 1
+set "PREFIX_NSIS=%PREFIX%\NSIS"
+robocopy . "%PREFIX_NSIS%" /S /XF bld.bat Docs Examples
+if errorlevel 1 exit 1
 
 cd ..
 if errorlevel 1 exit 1
 
 %PYTHON% "%RECIPE_DIR%\get_plugins.py"
 if errorlevel 1 exit 1
 
-copy "UAC\Plugins\x86-unicode\UAC.dll " "%PREFIX%\Plugins\x86-unicode\"
+copy "elevate\bin.x86-32\elevate.exe" "%PREFIX_NSIS%\Plugins\x86-unicode\"
 if errorlevel 1 exit 1
-copy "Untgz\untgz\unicode\untgz.dll" "%PREFIX%\Plugins\x86-unicode\"
+copy "UAC\U\UAC.dll" "%PREFIX_NSIS%\Plugins\x86-unicode\"
 if errorlevel 1 exit 1
-copy "UnicodePathTest_1.0\Plugin\UnicodePathTest.dll" "%PREFIX%\Plugins\x86-unicode\"
+copy "untgz\Plugins\x86-unicode\untgz.dll" "%PREFIX_NSIS%\Plugins\x86-unicode\"
 if errorlevel 1 exit 1
-
-copy "%SRC_DIR%\COPYING" + "UAC\License.txt" + "Untgz\untgz\LICENSE.TXT" "LICENSE.txt"
+copy "UnicodePathTest\Plugin\UnicodePathTest.dll" "%PREFIX_NSIS%\Plugins\x86-unicode\"
 if errorlevel 1 exit 1
 
+(
+type "%SRC_DIR%\COPYING"||exit 1
+
+echo(||exit 1
+echo(=== UAC plugin license information ===||exit 1
+type "UAC\License.txt"||exit 1
+
+echo(||exit 1
+echo(=== untgz plugin license information ===||exit 1
+type "untgz\LICENSE.TXT"||exit 1
+
+echo(||exit 1
+echo(=== UnicodePathTest plugin license information ===||exit 1
+type "UnicodePathTest\Readme.txt"||exit 1
```

Comment:
Yeah I guess that works. They at least say LGPL. Still its not a lot to go on. Thanks for adding it though.

Reply:
Ah, yeah, forgot to comment on that. But yes, you got it right, this was the only place I could find that mentioned a license. Not optimal but better than nothing, I suppose.